R/arcadia.R

```diff
@@ -69,7 +69,9 @@ theme_arcadia <- function(x_axis_type = "numerical", y_axis_type = "numerical",
     # tick specifications
     axis.ticks.x = x_axis_ticks,
     axis.ticks.y = y_axis_ticks,
-    axis.line = ggplot2::element_line(color="black", size = 0.5),
+    axis.ticks.length.x = ggplot2::unit(0.07, "in"), # 5 pixels is roughly 0.05 inches
```

Comment:
Just curious, is the comment or the value out of date?

```suggestion
    axis.ticks.length.x = ggplot2::unit(0.07, "in"), # 5 pixels is roughly 0.07 inches
```

Reply:
oh yes, thanks for catching!